plugins/inputs/apache/apache.go

```diff
@@ -15,13 +16,25 @@ import (
 )
 
 type Apache struct {
-	Urls []string
+	Urls          []string
+	useHTTPAuth   bool
+	Username      string
+	Password      string
+	Sslskipverify bool
+	Timeout       int
```

Comment:
Timeout should be an `internal.Duration` type, which can be represented in TOML like `timeout = "5s"`. You then also don't need to convert to a time object here, you can just use `a.Timeout.Duration`.